packages/compass-components/src/components/accordion.tsx

```diff
@@ -12,7 +12,9 @@ const buttonStyles = css({
   fontWeight: 'bold',
```

Comment:
Changes in this file:
- realign the chevron to the top, make sure the accordion looks right when the text breaks to multiple lines